routing/bandwidth.go

```diff
@@ -201,7 +195,13 @@ func (b *bandwidthManager) availableChanBandwidth(channelID uint64,
 		return 0, false
 	}
 
-	return b.getBandwidth(shortID, amount), true
+	bandwidth, err := b.getBandwidth(shortID, amount)
+	if err != nil {
+		log.Warnf("failed to get bandwidth for channel %v: %v",
+			shortID, err)
```

Comment:
we could return `0, true` here to make the code more explicit

Reply:
done